gammapy/cube/psf_map.py

```diff
@@ -331,3 +333,46 @@ def stack(self, other):
     def copy(self):
         """Copy PSFMap"""
         return deepcopy(self)
+
+    def sample_coord(psf_map, map_coord, random_state=0):
+        """ Apply PSF corrections on the coordinates of a set of simulated events.
+        psf_map : `~gammapy.cube.psf_map.PSFMap`.
```

Comment:
Please move the descriptions of the parameters in a separate `Parameters` section like so:
```
Parameters
----------
map_coord : `~gammapy.maps.MapCoord`
    Sequence of coordinates and energies of sampled events
....
```


Please also remove the documentation of `psf_map` once it's renamed to `self`.